spec/lib/aerospike/peers/parse_spec.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+RSpec.describe Aerospike::Peers::Parse do
```

Comment:
Can you please move the specs under `spec/lib/aerospike` to just `spec/aerospike` instead?